eframe/src/native/run.rs

```diff
@@ -424,12 +437,15 @@ mod glow_integration {
         native_options: &epi::NativeOptions,
         app_creator: epi::AppCreator,
     ) {
-        let event_loop = glutin::event_loop::EventLoopBuilder::with_user_event().build();
-        let glow_eframe = GlowWinitApp::new(&event_loop, app_name, native_options, app_creator);
-
         if native_options.run_and_return {
-            run_and_return(event_loop, glow_eframe);
+            with_event_loop(|event_loop| {
+                let glow_eframe =
+                    GlowWinitApp::new(event_loop, app_name, native_options, app_creator);
+                run_and_return(event_loop, glow_eframe);
+            });
```

Comment:
```suggestion
            with_event_loop(|event_loop| {
                {
                    let glow_eframe =
                        GlowWinitApp::new(event_loop, app_name, native_options, app_creator);
                    run_and_return(event_loop, glow_eframe);
                }

                // rerun to clean the event_loop
                use winit::platform::run_return::EventLoopExtRunReturn as _;
                event_loop.run_return(|_, _, control_flow| {
                    control_flow.set_exit();
                });
            });
```

Reply:
We should rerun to clean the event_loop after GlowWinitApp dropped.

Reply:
This works on my Windows 10 PC.

Reply:
I pushed it, but it doesn't help on my Mac :(